WTWSMS/history/provinces/100 - Avranches.txt

```diff
@@ -22,8 +22,6 @@ religion = celtic_pagan
 terrain = forest
 # History
 1.1.1 = { # Viking takeover
-	remove_settlement = b_caen
-	remove_settlement = b_bayeux
```

Comment:
It could also be that this is intended but badly explained. Sometimes the comment aren't always indicative as it is copy/paste from vanilla.

On bookmarks, have you been following the mod for a long time? Because our plans haven't changed, it is just that we need to finalise the goals for the full release before expanding history.